### DISCUSSION TITLE
Lived deprecated

### DISCUSSION THREAD

**Discussion by UsboKirishima** [UsboKirishima/lived#3] [Announcements]
This program may not work with your account, ClasseViva recently fixed its login, so this tool is now broken. 
Maybe in the future I will be able to bypass this problem by reviving the tool.

*- UsboKirishima*

**Comment by 47PADO47** [comment]
Non mi ero accorto che in tutte le richieste HTTP veniva incluso un body, anche in quelle GET. Ho pubblicato una nuova versione, in teoria dovrebbe funzionare tutto come prima.
